pkg/cmd/flags/policy.go

```diff
@@ -0,0 +1,219 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/filterscope"
+	"github.com/aquasecurity/tracee/types/policy"
+)
+
+// receive array of policies
+func PrepareFilterScopesForPolicies(policies []policy.Policy) (*filterscope.FilterScopes, error) {
+	parsedMap := map[int][]*filterFlag{}
+
+	// max 64?
+	for i, p := range policies {
+		err := validatePolicy(p)
+		if err != nil {
+			return nil, err
+		}
+
+		filterFlags := make([]*filterFlag, 0)
+
+		if p.Scope == nil {
+			return nil, errfmt.Errorf("scope cannot be empty")
+		}
+		// scope
+		for _, s := range p.Scope {
+			if s == "global" {
+				continue
```

Comment:
```suggestion
				break
```

Since `global` must be the unique scope when defined.